tests/rbd/test_rbd.py

```diff
@@ -31,8 +31,7 @@ def one_time_setup(node, rhbuild, branch: str) -> None:
     """
     try:
         node.exec_command(cmd="ls ceph")
-
-        return
+        node.exec_command(cmd="rm -rf ceph")
     except BaseException:  # noqa
         pass
 
```

Comment:
The try except block is not required as the plan is to remove the folder if found. Hence, we could avoid the block and have `node.exec_command(cmd="rm -rf ceph")`